### ISSUE TITLE
extra fields in json retrieval

### ISSUE THREAD

**Issue by tornaria** [roed314/seminars#834]
The json retrieval from #11 is great. I use it to prepare a mail reminder of talks with up to date information. I'm missing two fields which I hope would be easy to add:
- the link to the actual talk in mathseminars.org (https://researchseminars.org/talk/$SEMINAR/$ID/)
- the link to the ics for the talk

In fact the second is just the first one appended "ics" so just the first one is enough.

**Comment by edgarcosta** [comment]
@tornaria, maybe I misunderstand what you are asking, but I don't think it makes sense to add fields that can be trivially deduced from other fields.
The API gives seminar_id and seminar_ctr, with those you can easily build
- https://researchseminars.org/talk/$seminar_id/$seminar_ctr/
- https://researchseminars.org/talk/$seminar_id/$seminar_ctr/ics

**Comment by tornaria** [comment]
I mean the json retrieval from #11, which is separate from the API. I tried the API to see if I can switch to that, but I couldn't make it work (#833).


See below:
```
$ curl -s https://researchseminars.org/seminar/MathOnlineHostingEvents/json | jq
[
  {
    "speaker": "Juliette Bruce | David Eisenbud | Daniel Litt | Kevin McCurley | Akil Narayan | Isabel Vogt",
    "video_link": "https://www.youtube.com/watch?v=RLWr0zp9K5Q",
    "slides_link": "http://math.mit.edu/~drew/MathOnline20May2020PollResults.html",
    "title": "Panel discussion on hosting virtual events",
    "room": "",
    "comments": "",
    "abstract": "Many of us are now facing the daunting prospect of migrating mathematical events from a physical venue to a virtual environment. This presents many challenges, but also offers some new opportunities.  Join us for a panel discussion with conference organizers who have already confronted this task:\n\n<ul>\n\t<li><a href=\"https://juliettebruce.github.io/\">Juliette Bruce</a> (University of Wisconson-Madison), co-organizer of <a href=\"https://juliettebruce.github.io/cazoom/\">CAZoom - Virtually bringing together commutative algebra and related fields</a>.</li>\n    <li><a href=\"https://www.msri.org/~de/\">David Eisenbud</a> (Berkeley), director of the <a href=\"https://www.msri.org/web/cms\">Mathematical Sciences Research Institute</a>.</li>\n\t<li><a href=\"https://www.daniellitt.com/\">Daniel Litt</a> (University of Georgia), organizer of <a href=\"https://www.daniellitt.com/agonize\">Arithmetic Geometry is Online In Zoom Everyone (AGONIZE)</a>, co-organizer of <a href=\"https://sites.math.washington.edu/~jarod/wagon.html\">WAGON</a>.</li>\n    <li><a href=\"https://eurocrypt.iacr.org/2020/contact.php\">Kevin McCurley</a>, co-administrator of <a href=\"https://eurocrypt.iacr.org/2020/\">Eurocrypt 2020</a>.</li>\n\t<li><a href=\"http://www.sci.utah.edu/~akil/\">Akil Narayan</a> (University of Utah), co-organizer of <a href=\"https://icerm.brown.edu/programs/sp-s20/\">Model and dimension reduction in uncertain and dynamic systems</a>.</li>\n\t<li><a href=\"https://web.stanford.edu/~vogti/\">Isabel Vogt</a> (Stanford University), co-organizer of <a href=\"https://sites.math.washington.edu/~jarod/wagon.html\">Western Algebraic Geometry ONline (WAGON)</a>.</li>\n</ul>\n\n<b>Hosts</b>: <a href=\"https://math.mit.edu/~drew/\">Andrew Sutherland</a> (MIT) and <a href=\"http://sites.math.washington.edu/~bviray/\">Bianca Viray</a> (University of Washington).",
    "start_time": "2020-05-20 14:00:00-04:00",
    "end_time": "2020-05-20 15:00:00-04:00",
    "speaker_affiliation": "",
    "speaker_homepage": "https://juliettebruce.github.io/ | https://www.msri.org/~de/ | https://www.daniellitt.com/ | https://eurocrypt.iacr.org/2020/contact.php | http://www.sci.utah.edu/~akil/ | https://web.stanford.edu/~vogti/",
    "language": "en",
    "deleted": false,
    "paper_link": "http://math.mit.edu/~drew/MathOnline20May2020Questions.html",
    "stream_link": ""
  }
]
```

The field `seminar_ctr` is not there, nor anything that could lead to it.

**Comment by edgarcosta** [comment]
I have fixed #833, let me know if it also addresses this issue

**Comment by tornaria** [comment]
Thanks for your help, it was trivial to migrate my code to the api which gives me all the fields.

For the record, the json retrieval I did in my last comment can be replaced by
```
$ curl -s "https://researchseminars.org/api/0/lookup/series?series_id=\"MathOnlineHostingEvents\"" | jq .talks
[
  {
    "display": true,
    "speaker": "Juliette Bruce | David Eisenbud | Daniel Litt | Kevin McCurley | Akil Narayan | Isabel Vogt",
    "online": true,
    "video_link": "https://www.youtube.com/watch?v=RLWr0zp9K5Q",
    "slides_link": "http://math.mit.edu/~drew/MathOnline20May2020PollResults.html",
    "title": "Panel discussion on hosting virtual events",
    "room": "",
    "timezone": "America/New_York",
    "comments": "",
    "abstract": "Many of us are now facing the daunting prospect of migrating mathematical events from a physical venue to a virtual environment. This presents many challenges, but also offers some new opportunities.  Join us for a panel discussion with conference organizers who have already confronted this task:\n\n<ul>\n\t<li><a href=\"https://juliettebruce.github.io/\">Juliette Bruce</a> (University of Wisconson-Madison), co-organizer of <a href=\"https://juliettebruce.github.io/cazoom/\">CAZoom - Virtually bringing together commutative algebra and related fields</a>.</li>\n    <li><a href=\"https://www.msri.org/~de/\">David Eisenbud</a> (Berkeley), director of the <a href=\"https://www.msri.org/web/cms\">Mathematical Sciences Research Institute</a>.</li>\n\t<li><a href=\"https://www.daniellitt.com/\">Daniel Litt</a> (University of Georgia), organizer of <a href=\"https://www.daniellitt.com/agonize\">Arithmetic Geometry is Online In Zoom Everyone (AGONIZE)</a>, co-organizer of <a href=\"https://sites.math.washington.edu/~jarod/wagon.html\">WAGON</a>.</li>\n    <li><a href=\"https://eurocrypt.iacr.org/2020/contact.php\">Kevin McCurley</a>, co-administrator of <a href=\"https://eurocrypt.iacr.org/2020/\">Eurocrypt 2020</a>.</li>\n\t<li><a href=\"http://www.sci.utah.edu/~akil/\">Akil Narayan</a> (University of Utah), co-organizer of <a href=\"https://icerm.brown.edu/programs/sp-s20/\">Model and dimension reduction in uncertain and dynamic systems</a>.</li>\n\t<li><a href=\"https://web.stanford.edu/~vogti/\">Isabel Vogt</a> (Stanford University), co-organizer of <a href=\"https://sites.math.washington.edu/~jarod/wagon.html\">Western Algebraic Geometry ONline (WAGON)</a>.</li>\n</ul>\n\n<b>Hosts</b>: <a href=\"https://math.mit.edu/~drew/\">Andrew Sutherland</a> (MIT) and <a href=\"http://sites.math.washington.edu/~bviray/\">Bianca Viray</a> (University of Washington).",
    "seminar_id": "MathOnlineHostingEvents",
    "seminar_ctr": 1,
    "start_time": "2020-05-20 14:00:00-04:00",
    "end_time": "2020-05-20 15:00:00-04:00",
    "speaker_email": "",
    "speaker_affiliation": "",
    "speaker_homepage": "https://juliettebruce.github.io/ | https://www.msri.org/~de/ | https://www.daniellitt.com/ | https://eurocrypt.iacr.org/2020/contact.php | http://www.sci.utah.edu/~akil/ | https://web.stanford.edu/~vogti/",
    "topics": [
      "math"
    ],
    "language": "en",
    "deleted": false,
    "edited_at": "2020-05-25 17:14:43.969668-04:00",
    "paper_link": "http://math.mit.edu/~drew/MathOnline20May2020Questions.html",
    "stream_link": "",
    "deleted_with_seminar": false,
    "access_control": 1,
    "access_time": 1440,
    "access_hint": "",
    "access_registration": "",
    "by_api": false,
    "audience": 1,
    "chat_link": null
  }
]
```
which seems backwards compatible (I think it includes all the same fields, and more).
